test/e2e/apibinding/apibinding_test.go

```diff
@@ -0,0 +1,217 @@
+	require.NoError(t, err)
+
+	cfg := server.DefaultConfig(t)
+
+	kcpClients, err := clientset.NewClusterForConfig(cfg)
+	require.NoError(t, err, "failed to construct kcp cluster client for server")
+
+	dynamicClients, err := dynamic.NewClusterForConfig(cfg)
+	require.NoError(t, err, "failed to construct dynamic cluster client for server")
+
+	t.Logf("Creating source workspace in org lcluster %s", orgClusterName)
+	orgKcpClient := kcpClients.Cluster(orgClusterName)
+	sourceWorkspace := &tenancyv1alpha1.ClusterWorkspace{
+		ObjectMeta: metav1.ObjectMeta{
+			GenerateName: "source-",
+		},
+	}
+	sourceWorkspace, err = orgKcpClient.TenancyV1alpha1().ClusterWorkspaces().Create(ctx, sourceWorkspace, metav1.CreateOptions{})
+	require.NoError(t, err, "error creating source workspace")
+	require.Eventually(t, func() bool {
+		ws, err := orgKcpClient.TenancyV1alpha1().ClusterWorkspaces().Get(ctx, sourceWorkspace.Name, metav1.GetOptions{})
+		require.NoError(t, err, "error getting source workspace")
+		return ws.Status.Phase == tenancyv1alpha1.ClusterWorkspacePhaseReady
+	}, wait.ForeverTestTimeout, 100*time.Millisecond, "error waiting for source workspace to be ready")
+
+	server.Artifact(t, func() (runtime.Object, error) {
+		return orgKcpClient.TenancyV1alpha1().ClusterWorkspaces().Get(ctx, sourceWorkspace.Name, metav1.GetOptions{})
+	})
+
+	t.Logf("Creating target workspace in org lcluster %s", orgClusterName)
+	targetWorkspace := &tenancyv1alpha1.ClusterWorkspace{
+		ObjectMeta: metav1.ObjectMeta{
+			GenerateName: "target-",
+		},
+	}
+	targetWorkspace, err = orgKcpClient.TenancyV1alpha1().ClusterWorkspaces().Create(ctx, targetWorkspace, metav1.CreateOptions{})
+	require.NoError(t, err, "error creating target workspace")
+	require.Eventually(t, func() bool {
+		ws, err := orgKcpClient.TenancyV1alpha1().ClusterWorkspaces().Get(ctx, targetWorkspace.Name, metav1.GetOptions{})
+		require.NoError(t, err, "error getting target workspace")
+		return ws.Status.Phase == tenancyv1alpha1.ClusterWorkspacePhaseReady
+	}, wait.ForeverTestTimeout, 100*time.Millisecond, "error waiting for target workspace to be ready")
+
+	server.Artifact(t, func() (runtime.Object, error) {
+		return orgKcpClient.TenancyV1alpha1().ClusterWorkspaces().Get(ctx, targetWorkspace.Name, metav1.GetOptions{})
+	})
+
+	var (
+		sourceWorkspaceClusterName = org + ":" + sourceWorkspace.Name
+		targetWorkspaceClusterName = org + ":" + targetWorkspace.Name
+	)
+
+	t.Logf("Install a cowboys APIResourceSchema into workspace %q", sourceWorkspace.Name)
+	mapper := restmapper.NewDeferredDiscoveryRESTMapper(memory.NewMemCacheClient(kcpClients.Cluster(sourceWorkspaceClusterName).Discovery()))
+	err = helpers.CreateResourceFromFS(ctx, dynamicClients.Cluster(sourceWorkspaceClusterName), mapper, "apiresourceschema_cowboys.yaml", testFiles)
+	require.NoError(t, err)
+
+	t.Logf("Create an APIExport for it")
+	cowboysAPIExport := &apisv1alpha1.APIExport{
```

Comment:
(No action required) I should have started with the test, this makes the intention behind export+binding obvious to me.